server/channels/app/team_test.go

```diff
@@ -634,10 +676,6 @@ func TestPermanentDeleteTeam(t *testing.T) {
 	})
 	require.Nil(t, err, "Should create a team")
 
-	defer func() {
-		th.App.PermanentDeleteTeam(th.Context, team)
-	}()
-
```

Comment:
It would be a bit cleaner to assign the team created in L700 to a new variable and keep this cleanup code.

Not a blocker, though.